marklogic-unit-test-client/src/main/java/com/marklogic/test/unit/JUnitTestCase.java

```diff
@@ -49,11 +71,31 @@ public double getTime() {
 		return time;
 	}
 
-	public List<JUnitTestFailure> getTestFailures() {
+  public int getTests() {
+    return tests;
+  }
+
+  public int getSuccess() {
+    return success;
+  }
+
+  public int getFail() {
+    return fail;
+  }
+
+  public List<JUnitTestFailure> getTestFailures() {
 		return testFailures;
 	}
 
 	public void setTestFailures(List<JUnitTestFailure> testFailures) {
 		this.testFailures = testFailures;
 	}
+
+  public List<JUnitTestCoverage> getTestCoverages() {
+    return testCoverages;
+  }
+
+  public void setTestCoverages(List<JUnitTestCoverage> testCoverages) {
```

Comment:
Remove this as it's not needed (it looks like `setTestFailures` isn't needed either, but let's keep that as-is to minimize the changes in this PR).

Reply:
removed